From: Daniel Mack <daniel@zonque.org>
Date: Mon, 9 Mar 2015 18:00:46 +0100
Subject: [PATCH] Documentation: kdbus: fix location for generated files

The generated files should reside in Documentation/kdbus, not in the
top-level of the source tree. Also add a .gitignore file and ignore
everything that was built from the XML files.

Signed-off-by: Daniel Mack <daniel@zonque.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Documentation/kdbus/.gitignore | 2 ++
 Documentation/kdbus/Makefile   | 4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)
 create mode 100644 Documentation/kdbus/.gitignore

diff --git a/Documentation/kdbus/.gitignore b/Documentation/kdbus/.gitignore
new file mode 100644
index 000000000000..b4a77ccba9b4
--- /dev/null
+++ b/Documentation/kdbus/.gitignore
@@ -0,0 +1,2 @@
+*.7
+*.html
diff --git a/Documentation/kdbus/Makefile b/Documentation/kdbus/Makefile
index cd6b48ee41bf..f6d491251c25 100644
--- a/Documentation/kdbus/Makefile
+++ b/Documentation/kdbus/Makefile
@@ -18,10 +18,10 @@ HTMLFILES := $(patsubst %.xml, %.html, $(XMLFILES))
 XMLTO_ARGS := -m $(obj)/stylesheet.xsl
 
 %.7: %.xml
-	xmlto man $(XMLTO_ARGS) -o . $<
+	xmlto man $(XMLTO_ARGS) -o $(obj) $<
 
 %.html: %.xml
-	xmlto html-nochunks $(XMLTO_ARGS) -o . $<
+	xmlto html-nochunks $(XMLTO_ARGS) -o $(obj) $<
 
 mandocs: $(MANFILES)
 
